src/main/java/com/targomo/client/api/request/StatisticsRequest.java

```diff
@@ -86,26 +93,27 @@ public StatisticsResponse get(StatisticMethod method) throws TargomoClientExcept
 		// but only once
 		catch ( ProcessingException exception ) {
 
-			LOGGER.error(String.format("Executing statistics request (%s) to URI: '%s'", method.getPath(), target.getUri()), exception);
+			LOGGER.error("Error executing statistics request ({}) to URI: '{}'", path, target.getUri(), exception);
 
-			target = client.target(travelOptions.getFallbackServiceUrl()).path(method.getPath())
+			target = client.target(travelOptions.getFallbackServiceUrl()).path(path)
 					.queryParam("key", travelOptions.getServiceKey());
 
-			LOGGER.debug(String.format("Executing statistics request (%s) to URI: '%s'", method.getPath(), target.getUri()));
+			LOGGER.debug("Executing statistics request ({}) to URI: '{}'", path, target.getUri());
 
 			// Execute POST request
 			response = target.request().post(entity);
 		}
 
 		long roundTripTime = System.currentTimeMillis() - requestStart;
 
-		return validateResponse(response, requestStart, roundTripTime);
+		return responseValidator.validateResponse(response, requestStart, roundTripTime);
 	}
 
-	public static void main(String[] args) throws TargomoClientException, JSONException {
+	public static void main(String[] args) throws TargomoClientException {
```

Comment:
why is there a main method here? should this not be in the tests rather?

Reply:
I am not sure why tbh. and yes it was there before
Maybe it is made for the user of the java client to try out to call the API himself...